userguide/recommender/custom-model-ios.md

```diff
@@ -0,0 +1,50 @@
+# Custom Recommender Model for iOS Applications
+
+The Turi Create Recommender Model is also available for use in your iOS/macOS
```

Comment:
Note first and foremost that this is a beta feature (ask Aaron for wording to match the other places we call out beta-only features). For now, the user guide is dual-purpose, shared across 4.x and 5.x.